### PR TITLE
fix: api using db instead of mocks for api tests

### PR DESCRIPTION
**Pull request by vasco-santos**
Partially resolves https://github.com/web3-storage/web3.storage/issues/717 by using Docker to run a DB instead of mocks for api tests

Also removes remaining unused mocks from graphql API used when mocking fauna.

### CODE REVIEW

File: packages/api/test/user.spec.js [hunk 1]
```diff
@@ -26,7 +27,7 @@ describe('GET /user/account', () => {
     })
     assert(res.ok)
     const data = await res.json()
-    assert(data.usedStorage)
+    assert.strictEqual(data.usedStorage, 0)
```
Comment (vasco-santos) [inline packages/api/test/user.spec.js:30]: we previously returned a mock with a value that was not 0

File: packages/api/test/user.spec.js [hunk 2]
```diff
@@ -113,7 +114,7 @@ describe('POST /user/tokens', () => {
 describe('DELETE /user/tokens/:id', () => {
   it('error if not authenticated with magic.link', async () => {
     const token = await getTestJWT()
-    const res = await fetch(new URL('user/tokens/xyz', endpoint), {
+    const res = await fetch(new URL('user/tokens/2', endpoint), {
```
Comment (vasco-santos) [inline packages/api/test/user.spec.js:117]: older value failed because param was not a bigint

File: packages/api/test/user.spec.js [hunk 3]
```diff
@@ -304,7 +215,7 @@ describe('DELETE /user/uploads/:cid', () => {
 
   it('removes an upload', async () => {
     const token = 'test-magic'
-    const res = await fetch(new URL('user/uploads/bafybeibq5kfbnbvjgjg6bop4anhhaqopkc7t6mp2v3er3fkcv6ezhgvavg', endpoint), {
+    const res = await fetch(new URL('user/uploads/bafkreiajkbmpugz75eg2tmocmp3e33sg5kuyq2amzngslahgn6ltmqxxfa', endpoint), {
```
Comment (vasco-santos) [inline packages/api/test/user.spec.js:218]: Needs to be inserted before

**Review by alanshaw — APPROVED** [review]
🚀

**Pull request merged**